### PR TITLE
Create MATIO-1.5.21-GCCcore-10.2.0.eb

### PR DESCRIPTION
**Pull request by levlafayette**
New version, new compiler, new binutils

### CODE REVIEW

File: easybuild/easyconfigs/m/MATIO/MATIO-1.5.21-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+version = '1.5.21'
+
+homepage = 'https://sourceforge.net/projects/matio/'
+description = """matio is an C library for reading and writing Matlab MAT files."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_ZIP]
+checksums = ['bf1808a20412534a7c9e69c6eb0d2612']
```
Comment (Micket) [inline easybuild/easyconfigs/m/MATIO/MATIO-1.5.21-GCCcore-10.2.0.eb:21]: not a sha256sum?

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/MATIO/MATIO-1.5.21-GCCcore-10.2.0.eb (hunk 1 above)
Comment (levlafayette) [reply easybuild/easyconfigs/m/MATIO/MATIO-1.5.21-GCCcore-10.2.0.eb:21]: Whoops, that looks like a copy/paste error.

It should be:

2ac1d5406e51f2bdfaa545a535d6dbf6347be89891a06323961d7a2acbf37be9

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/3610309b9dca3edf5719b77b2ddef69f for a full test report.

**Comment by Micket** [comment]
Going in, thanks @levlafayette!

**Pull request merged**